reactDashboard/src/components/SingleValue/SingleValueBar.css

```diff
@@ -59,31 +82,67 @@
     flex-direction: row;
     flex-wrap: wrap;
   }
+  .region-single-value-container .single-value-heading {
+    margin-top: 10px;
+    margin-left: 20px;
+    height: 30%;
+  }
+  .region-single-value-container .single-value-number {
+    margin-left: 20px;
+  }
+  .region-single-value-container .single-value {
+    height: 175px;
+  }
   .single-value-container {
     width: 50%;
   }
 }
 
-@media screen and (max-width: 600px) {
+@media screen and (max-width: 640px) {
   .region-single-value-bar {
     flex-direction: column;
   }
   .region-single-value-container {
     width: 100%;
   }
+  .region-single-value-container .single-value {
+    height: 135px;
+  }
+  .region-single-value-container .single-value-heading {
+    height: 25%;
+  }
+}
+
+@media screen and (max-width: 575px) {
+  .single-value {
```

Comment:
For narrower screens, are the overview and regional single values now sharing the same dimensions? What is intended?

Reply:
This is intended - I have added a comment in the style sheet

Reply:
see above